src/styles/blocks/grid_container.scss

```diff
@@ -0,0 +1,27 @@
+.container {
+  --column-count: 1;
+  padding-top: 50px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  column-gap: 48px;
+  row-gap: 46px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
```

Comment:
Now everything works perfectly, but it is better not to change the value of the variable depending on the size, but to change the parameters themselves.